e2e/tests/odoo-openmrs-flows.spec.ts

```diff
@@ -119,7 +119,7 @@ test('Revising details of a synced OpenMRS drug order modifies the corresponding
   const drugOrderItem = await page.locator('table tbody td.o_data_cell:nth-child(3) span');
   await expect(drugOrderItem).toContainText('4.0 Tablet');
   await expect(drugOrderItem).toContainText('Twice daily - 5 day');
-  await expect(page.locator('td.o_data_cell:nth-child(9) span')).toHaveText('$ 14.88');
+  await expect(page.locator('td[name="price_subtotal"]')).toHaveText('$ 14.88');
```

Comment:
I am still not sure that this can be merged. Maybe we need to find a way to remove these taxes from the price calculations first so that the behaviour remains same.

@kdaud you don't need to check for price_subtotal if you where looking at total_price earlier and it was `$ 14.88` then even after upgrade of Odoo 17 it should match. The problem is not with the E2E tests but with the Odoo 17 deployment/implementation.